src/index.js

```diff
@@ -42,9 +43,15 @@ refreshBtn.addEventListener('click', async () => {
 
 submitBtn.addEventListener('click', async (event) => {
   event.preventDefault();
-  const name = nameInput.value;
-  const score = scoreInput.value;
-  await submitScores(gameId, name, score);
-  nameInput.value = '';
-  scoreInput.value = '';
+
+  if (nameInput.value === '' || scoreInput.value === '') {
+    errorMsg.style.display = 'flex';
+  } else {
+    const name = nameInput.value;
+    const score = scoreInput.value;
+    await submitScores(gameId, name, score);
+    nameInput.value = '';
```

Comment:
### Optional
- [ ] It would be nice if after entering the data(name, score) the score list updates the browser, however, after hitting the submit button the data is not seen in the browser only after we hit the refresh button. Kindly fix this bug